administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -232,13 +240,20 @@ COM_LOCALISE_TEXT_TRANSLATION_TRANSLATED="Translated"
 COM_LOCALISE_TEXT_TRANSLATION_UNCHANGED="Unchanged"
 COM_LOCALISE_TEXT_TRANSLATION_UNTRANSLATED="Untranslated"
 COM_LOCALISE_TITLE_TRANSLATION="Translation"
-COM_LOCALISE_TOOLTIP_TRANSLATION_INSERT="<strong>Insert</strong><br>Click to insert"
+COM_LOCALISE_TOOLTIP_TRANSLATION_EXTRA_KEYS_IN_TRANSLATION="If you do not want to keep a string in the translation, you should check the checkbox to remove it on 'Save'.<br> Some strings may not be in reference anymore.<br>Some strings may be specific plural forms that you need to keep."
+COM_LOCALISE_TOOLTIP_TRANSLATION_EXTRA_KEYS_IN_TRANSLATION_ICON="Checking the box this string will be removed from your translation."
 COM_LOCALISE_TOOLTIP_TRANSLATION_GOOGLE="<strong>Google</strong><br>Click to find a google translation"
+COM_LOCALISE_TOOLTIP_TRANSLATION_GRAMMAR_CASE="Grammar case\\nIf it is a typing error in the language 'en-GB' if you check the box, this changed text will not be shown in the other 'xx-XX' languages."
```

Comment:
I thought we would get rid of the  grammar case stuff and always use Modified values for en-GB when necessary. I'm confused.

Reply:
For me olny 2 options are posible on "modified values" with the "en-GB" language selected:

1. Deltete the check box from en-GB edition and keep the string as read only.
2. Use the check box for the only posible use: it is an en-GB grammar case than we can check as en-GB grammar case, also with the string as read-only.

Please, test it:
Go to en-GB edition and apply custom corrections on any string you can to edit there, due is allowed do it and "Save"
Then edit the same file in any other translation language and see the results of apply changes on en-GB reference language: all them are "fake" changes, "fake" changed text, etc.

Who will do it? No idea... but in this moment is allowed do it.

For me, all cases on normal edition at "en-GB"  view should be as "read-only" strings. if we access there we can revise, but not apply changes on our reference.